### DISCUSSION TITLE
google webcache broken with hash js files

### DISCUSSION THREAD

**Discussion by alexsanteenodev** [vercel/next.js#20419] [Help]
Hi! I have a problem with google webcache, seo optiomization.
When I first create website and deploy it to production, wait for google to cache my pages  - everything good.
But when I'm making changes, create a new build,  js/css files, chunks have new names with hash, and google webcache can't find old files and its broken.
Any suggestions?
My website [HTTPS://fee4bee.com](url) 
webcache: [cache:https://fee4bee.com/](url)

**Comment by azamanaza** [comment]
Also looking for a way to disable the hashing, esp in next13.

**Comment by alexsanteenodev** [comment]
@azamanaza Is it still a problem in the new next.js ? I haven't done recent updates in code, so I forgot about this issue.
